.github/workflows/merge.yml

```diff
@@ -84,6 +84,7 @@ jobs:
           echo JWT_KEY="${{secrets.JWT_KEY}}" >> .env
           echo API_URL=${{secrets.API_URL}} >> .env
           echo NEXT_PUBLIC_HCPATCHA_SITE_KEY=${{secrets.NEXT_PUBLIC_HCPATCHA_SITE_KEY}} >> .env
+          echo OPENAI_API_KEY=${{secrets.OPENAI_API_KEY}} >> .env
```

Comment:
This will need to be registered in the secrets. https://openai.com/blog/openai-api.

I'd estimate $1-2 per month with 1000 reviews per day. Comes with $18 free trial. If you add a card, make sure you set a hard spend limit just in case.